src/goto-harness/memory_snapshot_harness_generator.h

```diff
@@ -48,30 +49,92 @@ class memory_snapshot_harness_generatort : public goto_harness_generatort
   {
   }
 
+  /// 1. load memory table from the snapshot
```

Comment:
I'd start this documentation with a one-sentence summary. Otherwise I'd suspect this to render awkwardly.

Reply:
Done.